src/main/java/com/axonivy/market/entity/GithubRepoMeta.java

```diff
@@ -0,0 +1,20 @@
+package com.axonivy.market.entity;
+
+import static com.axonivy.market.constants.EntityConstants.GITHUB_REPO_META;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Document(GITHUB_REPO_META)
+public class GithubRepoMeta {
```

Comment:
`GithubRepoMeta` -> `GitHubRepoMeta`
Could you check other files that have `Github` and update them?